task-maker-format/src/ioi/statement/booklet.rs

```diff
@@ -207,7 +205,7 @@ impl Booklet {
                 packages.insert(package);
             }
             tasks.push(format!(
-                r"\input{{{}/statement.tex}}",
+                r"\subimport{{{}}}{{statement.tex}}",
```

Comment:
```suggestion
                r"\subimport{{./{}/}}{{statement.tex}}",
```

Apparently this fixes the compilation on Ubuntu! (and still works on arch)